pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/inv/RangeIndexCreator.java

```diff
@@ -165,26 +166,63 @@ public RangeIndexCreator(File indexDir, FieldSpec fieldSpec, FieldSpec.DataType
   }
 
   @Override
-  public void add(int dictId) {
-    _numberValueBuffer.put(_nextDocId, dictId);
-    _docIdBuffer.put(_nextDocId, _nextDocId);
-    _nextDocId = _nextDocId + 1;
+  public void add(int value) {
+    addValueToBuffer(value);
```

Comment:
Recommend inline these 2 methods for readability (as the existing code). Current way is not as readable

Reply:
done